socorro/processor/mozilla_transform_rules.py

```diff
@@ -1088,3 +1088,42 @@ def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
             processed_crash
         )
         return True
+
+
+#==============================================================================
+class ThemePrettyNameRule(Rule):
+    """The Firefox theme shows up commonly in crash reports referenced by its
+    internal ID. The ID is not easy to change, and is referenced by id in other
+    software.
+
+    This rule attempts to modify it to have a more identifiable name, like
+    other built-in extensions.
+
+    Must be run after the Addons Rule."""
+
+    #--------------------------------------------------------------------------
+    def __init__(self, config):
+        super(ThemePrettyNameRule, self).__init__(config)
+        self.theme_ext_id = "{972ce4c6-7e08-4474-a285-3208198ce6fd}"
+        self.pretty_ext_id = (
+            "{972ce4c6-7e08-4474-a285-3208198ce6fd} (default Firefox theme)"
+        )
```

Comment:
Let's make this class future-proof by using a dict instead. E.g.
```python
self.conversions = {
    '{972ce4c6-7e08-4474-a285-3208198ce6fd}': (
        '{972ce4c6-7e08-4474-a285-3208198ce6fd} (default Firefox theme)'
    ),
}
```
Who knows, there might be more of this. The class is name is generic (i.e. *not* called `DefaultFirefoxThemeNameRule`).